src/components/books/BooksList.js

```diff
@@ -7,8 +7,8 @@ const BooksList = () => {
     <ul>
       {books.map((book) => (
         <SingleBook
-          key={book.id}
-          id={book.id}
+          key={book.item_id}
+          id={book.item_id}
```

Comment:
- [ ] The `BooksList` component should show fetched books from the server, Data from the API requests should be stored correctly in Redux. Currently, it only shows added books from the redux store(same as the previous milestone) not the fetched books from the API.

Reply:
Done ✅